profile/README.md

```diff
@@ -3,12 +3,10 @@
 Welcome to the DAI course at [HEIG-VD](https://heig-vd.ch), Switzerland!
 
 ## Getting started
-
 Do not know where to start? Have a look at the
 [main repository for the course](https://github.com/heig-vd-dai-course/heig-vd-dai-course)!
 
 ## Teaching staff
-
```

Comment:
You shouldn't have removed the line between the title and the HTML comment. Maybe it's your IDE that did format the file for you? Please re-add the lines.